src/group_theory/perm/sign.lean

```diff
@@ -213,6 +215,15 @@ def support [fintype α] (f : perm α) : finset α := univ.filter (λ x, f x ≠
 @[simp] lemma mem_support [fintype α] {f : perm α} {x : α} : x ∈ f.support ↔ f x ≠ x :=
 by simp only [support, true_and, mem_filter, mem_univ]
 
+lemma support_mul_subset [fintype α] {f g : perm α} :
+  (f * g).support ⊆ f.support ∪ g.support :=
```

Comment:
The `support_pow_le` lemma below uses `≤` instead of `⊆`. Do you have a feeling for which you prefer? I suppose if we used `⊆`, then `∪` would be `\sup`...

Reply:
Let me merge in the `card_support_perm` branch, I guess. It sorts this out...

Reply:
Actually, that's a bad idea. Let me just copy the version of that lemma.

Reply:
Would it be better to just merge that branch first? What's the PR?

Reply:
It's #6951. I think this is the only overlap, though. I ended up not needing this lemma in this PR after all, I will delete it from this branch.